src/utils/constants.ts

```diff
@@ -31,6 +31,37 @@ export const UNIVERSAL_ROUTER_ADDRESS = (chainId: number): string => {
   }
 }
 
+export const UNIVERSAL_ROUTER_CREATION_BLOCK = (chainId: number): number => {
+  switch (chainId) {
+    case 1: // mainnet
+      return 17143817
+    case 5: // goerli
+      return 8940568
+    case 11155111: // sepolia
```

Comment:
we're repeating these chain numbers twice. we should consider just creating a mapping

Reply:
(and looks like we forgot to put WETH In for some of the chains)